package.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "13.4.0",
+  "version": "13.5.0",
```

Comment:
How certain are we that there no breaking change here when upgrading from 13.4.0 to 13.5.0?

I can't think of anything here that is breaking. If we have any doubts we should try upgrading from 13.4.0 to 13.5.0 to see if there are any issues.